src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,110 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * The class to write data to and retrieve data from a file.
+ */
+public class Storage {
+
+    /** the path of the file used for storage */
+    private String filePath;
+
+    /**
+     * The constructor of the Storage class
+     *
+     * @param filePath the path of the file used for storage
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * The method to load the data from an already existing file into a new list of tasks
+     *
+     * @return the list of tasks
+     * @throws DukeException
+     */
+    public ArrayList<Task> load() throws DukeException {
```

Comment:
I think something along the lines of load existing tasks would give more details